README.md

```diff
@@ -70,6 +78,8 @@ Add your `accessToken` in a script tag at the end of your body:
 </body>
 ```
 
+Mapbox [recommends](https://docs.mapbox.com/help/tutorials/first-steps-ios-sdk/#display-the-users-location) the explanation "Shows your location on the map and helps improve the map".
```

Comment:
This line actually belongs in line 58 after the `NSLocationWhenInUseUsageDescription` explanation.